src/main/java/step3/GameBoard.java

```diff
@@ -27,11 +29,16 @@ public Count registerTrialCount() {
         return INPUT_VIEW.inputValue();
     }
 
-    public void renderRaceProgress(RacingCarGroup carGroup) {
+    public void renderRaceResult(List<Record> recordList) {
+        recordList.forEach(r -> {
+            renderPosition(r);
+        });
```

Comment:
한 라인으로 끝나는 경우 중괄호는 생략이 가능합니다. 
```suggestion
record.recordList().stream()
                .forEach(p -> System.out.println(appendText(p.getPosition())));
```